owl/SOMA-OBJ.owl

```diff
@@ -787,6 +812,36 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasReferencePose -->
+
+    <owl:DatatypeProperty rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasReferencePose">
+        <rdfs:subPropertyOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#hasShapeParameter"/>
+        <rdfs:domain rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#ShapeRegion"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
```

Comment:
Can we refer to a space region instead?
I think there is e.g. `6DPose` which is a region.
Maybe this can be modeled as part or constituent?